### PR TITLE
Feat/op

### PR DESCRIPTION
**Pull request by MicaiahReid**

### CODE REVIEW

File: addons/evm/src/commands/actions/transfer_eth.rs [hunk 1]
```diff
@@ -0,0 +1,381 @@
...
+use crate::typing::EVM_ADDRESS;
+use txtx_addon_kit::constants::TX_HASH;
+
+use super::get_signer_did;
+
+lazy_static! {
+    pub static ref TRANSFER_ETH: PreCommandSpecification = define_command! {
+        TransverEth => {
+            name: "Coming soon",
+            matcher: "transfer_eth",
```
Comment (lgalabru) [inline addons/evm/src/commands/actions/transfer_eth.rs:36]: `send_eth` would be shorter? We went with `send_stx`, Stacks side.

File: addons/evm/src/signers/secret_key.rs [hunk 2]
```diff
@@ -325,6 +327,7 @@ impl SignerImplementation for EvmSecretKeySigner {
                 )
             })?;
             let tx_nonce = tx_envelope.nonce();
+            let tx_chain_id = tx_envelope.chain_id().unwrap_or(u64::MAX);
```
Comment (lgalabru) [inline addons/evm/src/signers/secret_key.rs:330]: I'm not sure I understand the rationale behind this `u64::MAX` fall back.

File: addons/ovm/examples/ovm.tx [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+        "ProtocolVersions": "0x6226b256912843C624B163b1CC9dB13D96894b03",
+        "ProtocolVersionsProxy": "0x1cA58C594C4Db979e0aaD6B821018074CBFa3d48",
+        "ProxyAdmin": "0xa869413806d4496BE149896d0c01f5b78e790d23",
+        "SafeProxyFactory": "0xa6B71E26C5e0845f74c812102Ca7114b6a896AB2",
+        "SafeSingleton": "0xd9Db270c1B5E3Bd161E8c8503c55cEABeE709552",
+        "SuperchainConfig": "0xdD6191EE7b7aeCca6f14Ef2bCF76183467D6ED0E",
+        "SuperchainConfigProxy": "0x3389EA8C39393D0073920EF23e9f4F73C3930637",
+        "SystemConfig": "0x5fEE6D8c69e309d636A40492CD10950fA9422E69",
+        "SystemConfigProxy": "0x677209694761A0B010c1C94503b0C310f0e6Bd21",
+        "SystemOwnerSafe": "0x44e5a74148BFAf7713Bf3bA050896982e69bBFea"
```
Comment (lgalabru) [inline addons/ovm/examples/ovm.tx:35]: Will developers have to provide all these addresses?

File: addons/ovm/examples/ovm.tx [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+    l1_rpc_kind = "infura"
+    sequencer_secret_key = "0xf044e9107ef8a741a2e958d03aa5e99a7c525c4eb74158917a80b70435de6198"
+    batcher_secret_key = "0xfe3d11cd8ccdfffd13130d4fdcd26e97db3ab4c9e043961c2923850b7034f4a1"
+    proposer_secret_key = "0xc51bcf8d6e1702999d5a2ed6e14baeb5cc1925df446457d86cb3ceca2d142a1f"
+}
+
+action "bridge_tokens" "evm::transfer_eth" {
+    description = "Bridge some tokens from L1 to L2"
+    recipient_address = "0x0C5C3aF1508278cbC536827a1cCb2f69195b7068"
+    amount = 1000000000000000000 // 1 eth
```
Comment (lgalabru) [inline addons/ovm/examples/ovm.tx:66]: Should we introduce a `evm::eth_to_wei()` or similar?

File: addons/evm/src/signers/secret_key.rs (hunk 2 above)
Comment (lgalabru) [reply addons/evm/src/signers/secret_key.rs:330]: oh I see.

File: addons/evm/src/commands/actions/transfer_eth.rs (hunk 1 above)
Comment (MicaiahReid) [reply addons/evm/src/commands/actions/transfer_eth.rs:36]: Done in a1c6d0b9d304d55f4b091535b2444fae3b5ec99c

File: addons/ovm/examples/ovm.tx (hunk 4 above)
Comment (MicaiahReid) [reply addons/ovm/examples/ovm.tx:66]: I've added this to the planner!

File: addons/ovm/examples/ovm.tx (hunk 3 above)
Comment (MicaiahReid) [reply addons/ovm/examples/ovm.tx:35]: For now, yes. These are the deployments from the L1 terraforming.

Eventually we will even have these be integrated as part of the command, or they'll be a standalone runbook that you can call.

File: addons/evm/src/signers/secret_key.rs (hunk 2 above)
Comment (MicaiahReid) [reply addons/evm/src/signers/secret_key.rs:330]: Very true, fixed! An `unwrap` is safe here - we always require the user provide a chain id, and it is always set on the transactions we build.

**Pull request merged**